src/scores.ts

```diff
@@ -377,29 +387,39 @@ function attributeWithPrefix(attribute: string): string {
   }
 }
 
-// Returns text description of the hideCommentReason to be displayed to the
-// user.
-export function getHideReasonDescription(hideCommentReason: AttributeScore): string {
-  if (hideCommentReason.score >= BLARING_THRESHOLD) {
-    return 'Blaring';
-  } else if (hideCommentReason.score >= LOUD_THRESHOLD) {
-    return 'Loud';
-  } else if (hideCommentReason.score >= MEDIUM_THRESHOLD) {
-    return 'Medium';
-  } else if (hideCommentReason.score >= LOW_THRESHOLD) {
-    return 'Low';
+// Returns text description of the CommentVisibilityDescription to be displayed
+// to the user.
+export function getHideReasonDescription(commentVisibility: CommentVisibilityDecision): string {
+  if (commentVisibility.kind === 'showComment') {
+    return '';
+  } else if (commentVisibility.kind === 'hideCommentDueToUnsupportedLanguage') {
+    return 'Tune doesn\'t currently support this language.';
   } else {
-    return 'Quiet';
+    assert(commentVisibility.kind === 'hideCommentDueToScores');
+    if (commentVisibility.scaledScore >= BLARING_THRESHOLD) {
+      return 'Blaring';
+    } else if (commentVisibility.scaledScore >= LOUD_THRESHOLD) {
+      return 'Loud';
+    } else if (commentVisibility.scaledScore >= MEDIUM_THRESHOLD) {
+      return 'Medium';
+    } else if (commentVisibility.scaledScore >= LOW_THRESHOLD) {
+      return 'Low';
+    } else {
+      return 'Quiet';
+    }
   }
 }
 
-export function getFeedbackQuestion(hideCommentReason: AttributeScore,
-                                    subtypesEnabled: boolean): string {
-  // TODO: do we want to use a different threshold here? is it okay to
-  // mention the attribute when the confidence is not particularly high?
-  if (subtypesEnabled && hideCommentReason.score >= LOUD_THRESHOLD) {
-    return 'Is this ' + attributeWithPrefix(hideCommentReason.attribute) + '?';
-  } else {
-    return 'Should this be hidden?';
+export function getFeedbackQuestion(commentVisibility: CommentVisibilityDecision,
+                                    subtypesEnabled: boolean)
+: string {
+  if (commentVisibility.kind === 'showComment') {
+    return '';
+  }
+  if (subtypesEnabled && commentVisibility.kind === 'hideCommentDueToScores') {
+    if (commentVisibility.scaledScore >= LOUD_THRESHOLD) {
```

Comment:
Fixed, thanks. I had it this way originally because I _thought_ the compiler gave an error that it couldn't determine that `commentVisibility` had the `scaledScore` field when it was just a single merged conditional, but it works now, so it must have been some other issue.